google-ads-examples/src/main/java/com/google/ads/googleads/examples/shoppingads/RejectMerchantCenterLink.java

```diff
@@ -0,0 +1,164 @@
+      // into the code here. See the parameter class definition above for descriptions.
+      params.customerId = Long.parseLong("INSERT_CUSTOMER_ID_HERE");
+      params.merchantCenterAccountId = Long.parseLong("INSERT_MERCHANT_CENTER_ACCOUNT_ID_HERE");
+    }
+
+    GoogleAdsClient googleAdsClient;
+    try {
+      googleAdsClient = GoogleAdsClient.newBuilder().fromPropertiesFile().build();
+    } catch (FileNotFoundException fnfe) {
+      System.err.printf(
+          "Failed to load GoogleAdsClient configuration from file. Exception: %s%n", fnfe);
+      return;
+    } catch (IOException ioe) {
+      System.err.printf("Failed to create GoogleAdsClient. Exception: %s%n", ioe);
+      return;
+    }
+
+    try {
+      new RejectMerchantCenterLink()
+          .runExample(googleAdsClient, params.customerId, params.merchantCenterAccountId);
+    } catch (GoogleAdsException gae) {
+      // GoogleAdsException is the base class for most exceptions thrown by an API request.
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
+      int i = 0;
+      for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()) {
+        System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+      }
+    }
+  }
+
+  /**
+   * Runs the example.
+   *
+   * @param googleAdsClient the Google Ads API client.
+   * @param customerId the client customer ID of the Google Ads account to reject the link request.
+   * @param merchantCenterAccountId the Merchant Center account ID for the account requesting to
+   *     link.
+   * @throws GoogleAdsException if an API request failed with one or more service errors.
+   */
+  private void runExample(
+      GoogleAdsClient googleAdsClient, long customerId, long merchantCenterAccountId) {
+
+    // Rejects a pending link request or unlinks an enabled link for a Google Ads account with
+    // customerId from a Merchant Center account with merchantCenterAccountId.
+    try (MerchantCenterLinkServiceClient merchantCenterLinkService =
+        googleAdsClient.getLatestVersion().createMerchantCenterLinkServiceClient()) {
+      ListMerchantCenterLinksResponse response =
+          merchantCenterLinkService.listMerchantCenterLinks(
+              ListMerchantCenterLinksRequest.newBuilder()
+                  .setCustomerId(Long.toString(customerId))
+                  .build());
+
+      System.out.printf(
+          "%d Merchant Center link(s) found with the following details:%n",
+          response.getMerchantCenterLinksCount());
+
+      for (MerchantCenterLink merchantCenterLink : response.getMerchantCenterLinksList()) {
+        System.out.printf(
+            "Link '%s' is in status '%s'.%n",
```

Comment:
nit: "is in status" sounds a bit strange. Maybe "has status" or "with status"?

Reply:
+1

Reply:
Agree. Will make the edit to both files using "has status".